resources/js/features/games/components/GameAchievementSetsContainer/GameAchievementSet/GameAchievementSetHeader/GameAchievementSetHeader.tsx

```diff
@@ -0,0 +1,84 @@
+import { type FC, useMemo } from 'react';
+import { Trans, useTranslation } from 'react-i18next';
+import { LuChevronDown } from 'react-icons/lu';
+
+import { WeightedPointsContainer } from '@/common/components/WeightedPointsContainer';
+import { cn } from '@/common/utils/cn';
+
+interface GameAchievementSetHeaderProps {
+  gameAchievementSet: App.Platform.Data.GameAchievementSet;
+  isOnlySetForGame: boolean;
+  isOpen: boolean;
+}
+
+export const GameAchievementSetHeader: FC<GameAchievementSetHeaderProps> = ({
+  gameAchievementSet,
+  isOnlySetForGame,
+  isOpen,
+}) => {
+  const { t } = useTranslation();
+
+  const { achievementSet, title } = gameAchievementSet;
+  const { achievements, imageAssetPathUrl } = achievementSet;
+
+  const { totalPoints, totalPointsWeighted } = useMemo(
+    () => getAchievementStats(achievements),
+    [achievements],
+  );
+
+  return (
+    <div className="w-full rounded bg-embed py-1.5 pl-2 pr-3">
+      <div className="flex items-center justify-between text-neutral-300 light:text-neutral-700">
+        <div className="flex items-center gap-3">
+          <img src={imageAssetPathUrl} width={52} height={52} className="rounded-sm" />
+
+          <div className="flex flex-col items-start gap-0">
+            <span>{title ?? t('Base Set')}</span>
+
+            <span className="text-xs text-text">
+              <Trans
+                i18nKey="{{achievementsCount, number}} achievements worth {{pointsCount, number}} <1>({{retroPointsCount, number}})</1> points"
+                values={{
+                  achievementsCount: achievements.length,
+                  pointsCount: totalPoints,
+                  retroPointsCount: totalPointsWeighted,
+                }}
+                components={{ 1: <WeightedPointsContainer /> }}
+              />
+            </span>
+          </div>
+        </div>
+
+        {!isOnlySetForGame ? (
+          <LuChevronDown
+            data-testid="chevron"
+            className={cn(
+              'size-5 transition-transform duration-300',
+              isOpen ? 'rotate-180' : 'rotate-0',
+            )}
+          />
+        ) : null}
```

Comment:
Changing the sort order while a set is collapsed causes it to refresh expanded, but with the button still in a collapse state.